skiko/src/awtMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.kt

```diff
@@ -41,6 +38,8 @@ internal class MetalRedrawer(
     private val contextHandler: MetalContextHandler
 
     companion object {
+        @OptIn(ExperimentalCoroutinesApi::class)
+        private val dispatcherToBlockOn = Dispatchers.IO.limitedParallelism(64)
```

Comment:
Why we name it "toBlockOn"? Sounds that we block something, but we don't block in coroutines, we suspend.

Reply:
in this case we actually use this dispatcher for waiting for vsync. Which means that thread from it will be blocked for some time. That the reason why we switch to `Dispatcher.IO`. The same is true for the others `dispatcherToBlockOn `